documentation/content/Supported Plugins/series-plugin.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: Displaying Multi-Part Articles - Part 1
+Tags: pelican-plugin, navigation, web-design
+Category: Supported Plugins
+Date: 2019-12-07
+Slug: how-to-use-series-plugin
+Comment_id: 3ws2xke-how-to-use-series-plugin
+Subtitle:
+Summary: Elegant can be configured to provide a series section on the right sidebar. Only visible in articles that are labelled as part of a series, this indicator allows navigation between the articles in the series.
+Keywords:
+Authors: Talha Mansoor, Jack De Winter
```

Comment:
I am not the author. You don't have to mention me unless I wrote something.

Reply:
Fair enough.  Since you wrote the multipart article originally, and the series article is over 90% that article, didn't feel correct to me to leave you off the list.  Your call.  Either way is fine.